File: SDKs/googleapis.com/vmmigration/v1alpha1/go/pkg/models/shared/listcutoverjobsresponse.go
package shared

// ListCutoverJobsResponse
// Response message for 'ListCutoverJobs' request.
type ListCutoverJobsResponse struct {
	CutoverJobs   []CutoverJob `json:"cutoverJobs,omitempty"`
	NextPageToken *string      `json:"nextPageToken,omitempty"`
	Unreachable   []string     `json:"unreachable,omitempty"`
}

File: SDKs/googleapis.com/vmmigration/v1/typescript/src/sdk/models/shared/listcutoverjobsresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { CutoverJob } from "./cutoverjob";



// ListCutoverJobsResponse
/** 
 * Response message for 'ListCutoverJobs' request.
**/
export class ListCutoverJobsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=cutoverJobs", elemType: CutoverJob })
  cutoverJobs?: CutoverJob[];

  @SpeakeasyMetadata({ data: "json, name=nextPageToken" })
  nextPageToken?: string;

  @SpeakeasyMetadata({ data: "json, name=unreachable" })
  unreachable?: string[];
}
